solver/src/main/java/org/chocosolver/solver/Solver.java

```diff
@@ -626,6 +626,15 @@ public void propagate() throws ContradictionException {
         }
     }
 
+    /**
+     * Return the minimum conflicting set from a conflicting set that is causing contradiction.
+     * @param conflictingSet
+     * @return minimumConflictingSet
+     */
+    public List<Constraint> findMinimumConflictingSet(List<Constraint> conflictingSet) {
```

Comment:
I think this is a good place to add assertions and eventually throw exception if looking for the MCS is called in inappropriate way (during solving for instance). The javadoc should also exhibit these assertions.

Reply:
Sure, I add the check for solving and created a unitary test to thist.